[MTM-59297] Rephrase release note. Skip completely information about … (#2278)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-472-0-skip-domain-attribute-in-authorization-set-cookie-header.md

ticket: MTM-59297
version: 10.20.472.0
---

> A security improvement in the way the platform manages the session cookies. The `domain` attribute in the cookie is made applicable strictly to the tenant accessed.

Reviewer: ```suggestion
Security has been improved by ensuring the `domain` attribute in a cookie is made applicable strictly to the tenant accessed.
```
